app/src/main/res/layout/fragment_media_details.xml

```diff
@@ -8,304 +8,286 @@
     android:orientation="vertical">
 
     <com.google.android.material.appbar.AppBarLayout
+        app:layout_constraintTop_toTopOf="parent"
         android:layout_width="match_parent"
+        android:id="@+id/layout_toolbar"
         android:layout_height="wrap_content">
 
         <androidx.appcompat.widget.Toolbar
             android:id="@+id/toolbar"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             app:navigationIcon="@drawable/ic_arrow_back_black_24dp"
-            app:title="@string/media_detail"
-            />
+            app:title="@string/media_detail" />
 
     </com.google.android.material.appbar.AppBarLayout>
 
     <ScrollView
-        android:nestedScrollingEnabled="false"
-        android:id="@+id/mediaDetailScrollView"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:background="@color/white"
-        android:clipToPadding="true"
-        android:fillViewport="true"
-        android:fitsSystemWindows="true"
-        >
-
-        <LinearLayout
+        android:layout_height="match_parent"
+        app:layout_constraintTop_toBottomOf="@id/layout_toolbar"
+        android:orientation="vertical">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
@gconnect can we avoid nesting in the layouts. Please take a look at this article https://medium.com/@vleonovs8/how-to-avoid-nested-containers-inside-constraintlayout-3a4d020edd0f